docker/iml-cli-proxy.sh

```diff
@@ -2,5 +2,9 @@
 
 trailer=$(docker service ps -f 'name=iml_iml-manager-cli.1' iml_iml-manager-cli -q --no-trunc | head -n1)
 TTYOPT=$(tty -s && echo "-t")
+ENVOPT=""
+if [ -n "$RUST_LOG" ]; then
+	ENVOPT="-e RUST_LOG=$RUST_LOG"
+fi
 
-docker exec -i $TTYOPT iml_iml-manager-cli.1.$trailer iml "${@:1}"
+docker exec -i $TTYOPT $ENVOPT iml_iml-manager-cli.1.$trailer /usr/bin/iml "${@:1}"
```

Comment:
do we need to specify full path here?

Reply:
It fixes the quote munging inside docker.

Reply:
Just trying this change out, I'm getting some odd behavior.  iml to /usr/bin/iml gave me a "relative URL without a base" error, the $ENVOPT gives me "ignoring '': invalid filter directive"

Reply:
I might be missing something, but #2294 removes the other path entirely.

Reply:
to do what you want here, you need docker exec -w /usr/bin -i ... iml ${@:1}"

Reply:
I guess the explicit full path for iml isn't required, but it doesn't hurt.

Reply:
If I run it by hand, it fails.  So yes, it does.